File: php/code/hello.php
<?php
// hello.php —— 第 1-2 章：第一个程序
// 放到虚拟机的 C:\phpstudy\WWW\php\ 下，浏览器访问 http://192.168.11.133/php/hello.php

echo 'Hello, PHP!<br>';

$name = '同学';
echo '欢迎，' . $name . '！<br>';

// 单引号 vs 双引号
echo '单引号（原样输出）：$name<br>';
echo "双引号（解析变量）：$name<br>";

// 变量与类型
$age    = 20;
$height = 1.75;
$isNew  = true;
var_dump($name, $age, $height, $isNew);

// 弱类型演示（第 3 章会细讲它的危险）
echo '<br>弱类型：0 == \'abc\' 的结果是 ';
var_dump(0 == 'abc');

// 环境信息
echo '<br>PHP 版本：' . PHP_VERSION;
// phpinfo();   // 需要时再打开，真实环境里它是信息泄露点

File: python/code/hello.py
"""
hello.py —— 第 2 章 · 第一个程序

怎么跑：  py hello.py
应该看到：欢迎语、解释器路径、Python 版本
安全等级：✅ 安全
"""

print("Hello, Python!")

name = "同学"
age = 20
print(f"欢迎，{name}！你今年 {age} 岁。")

# 三种引号
print('单引号：原样写，但要小心里面的单引号')
print("双引号：可以直接写英文单引号 ' ")
print("""三引号：可以写多行
第二行
第三行""")

# int 是任意精度，不会溢出
big = 2 ** 200
print("2 的 200 次方有", len(str(big)), "位数字 —— Python 的 int 不会溢出")

# 看看自己在哪个解释器里跑（排查环境问题的第一步）
import sys
print("解释器路径：", sys.executable)
print("版本：", sys.version.split()[0])

# 类型是可以查的
for v in [1, 1.5, "abc", True, None, [1], {"a": 1}]:
    print(f"{str(v):>12}  →  {type(v).__name__}")
